client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -204,6 +209,11 @@ const CopySiteItem = ( { recordTracks, site }: SitesMenuItemProps ) => {
 			onClick={ () => {
 				clearSignupDestinationCookie();
 				setPlanCartItem( { product_slug: plan.product_slug } );
+				purchases.map( ( purchase ) => {
+					if ( 'marketplace_plugin' === purchase.productType ) {
+						// setPlanCartItem( { product_slug: purchase.productSlug } );
```

Comment:
Not quite sure what's appropriate here, so I've asked shillings: p1674064554074069-slack-C0117V2PCAE

Reply:
I managed to hack a product into the cart! https://github.com/Automattic/wp-calypso/pull/72291/commits/258d7cf63f4b5a003b7024be3371b3d5358b00d9#diff-3a158217b3c977b514f4fe1c284c0fd2e8072619c0c29a80e7b7b7bee8bc3623R115-R117